testserver/tenant.go

```diff
@@ -201,27 +202,30 @@ func (ts *testServerImpl) NewTenantServer(proxy bool) (TestServer, error) {
 		"--kv-addrs=" + pgURL.Host,
 		"--sql-addr=" + sqlAddr,
 		"--http-addr=:0",
-	}
+	}}
 
 	tenant := &testServerImpl{
 		serverArgs: ts.serverArgs,
 		version:    ts.version,
 		state:      stateNew,
+		nodeStates: []int{stateNew},
```

Comment:
I'm thinking instead of making 4 seperated lists for `args`, `nodeStates`, `cmd` and `listeningURLFile`, can we wrap them into a struct to represent info for each node? 
I.e. can we do
```
testServerImpl{
nodeInfoList: []nodeInfo{node1, node2, node3},
...
}

type nodeInfo struct {
startArgs []string,
nodeState int,
cmd []*exec.Cmd,
listenningURLFile string,
}
```

Reply:
Done